### ISSUE TITLE
ITMS-90892: Missing recommended icon

### ISSUE THREAD

**Issue by shivanraptor** [TelegramMessenger/Telegram-iOS#674]
### Checklist
- [x] I am reporting an issue in existing functionality that does not work as intended
- [x] I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
After a successful build of the `release-8.2.2` source, when delivering the IPA to App Store via Trasnsporter, the App Store Connect said 3 recommended icons are missing.

### Expected Behavior
All icons should be included in the build.

### Actual Behavior
3 recommended icons are missing.

> ITMS-90892: Missing recommended icon - The bundle does not contain an alternate app icon for iPad of exactly '76x76' pixels, in .png format for iOS versions >= 7.0. To support older operating systems, the icon may be required in the bundle outside of an asset catalog. Make sure the Info.plist file includes appropriate entries referencing the file. See https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleicons
> 
> ITMS-90892: Missing recommended icon - The bundle does not contain an alternate app icon for iPad of exactly '167x167' pixels, in .png format for iOS versions supporting iPad Pro. To support older operating systems, the icon may be required in the bundle outside of an asset catalog. Make sure the Info.plist file includes appropriate entries referencing the file. See https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleicons
> 
> ITMS-90892: Missing recommended icon - The bundle does not contain an alternate app icon for iPad of exactly '152x152' pixels, in .png format for iOS versions >= 7.0. To support older operating systems, the icon may be required in the bundle outside of an asset catalog. Make sure the Info.plist file includes appropriate entries referencing the file. See https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleicons

### Steps to Reproduce
1. git clone the `release-8.2.2` tag
2. change the appropriate configurations according to README
3. build, sign and submit to App Store

### Environment
(probably unrelated)
**Device:** `iPhone Xr`
**iOS version**: `15.1` 
**App version:** `8.2.2`
**Xcode version:** `13.0`
**macOS version:** `11.3`
**bazel version:** `4.0.0`

**Comment by shivanraptor** [comment]
#576

**Comment by ali-fareed** [comment]
Could you try adding the missing icons?

**Comment by shivanraptor** [comment]
How to add those icons. I open the icon sets in Xcode 13.0, but I found no missing icon placeholder. Or I have to update some Bazel `BUILD` file? Thanks.

p.s. the App without these recommended icons can still pass the App Store approval process.

**Comment by shivanraptor** [comment]
Additionally, how do I set the iMessage icon? it looks distorted right now.

![Screenshot 2021-12-14 at 2 19 20 PM](https://user-images.githubusercontent.com/997890/145943975-874af60e-d5c9-404e-b03b-1526ea9b75dc.png)
.

**Comment by janeroseloving** [comment]
yes, I have this issue, 8.1.2 is ok, but 8.3.0 show 'ITMS-90892: Missing recommended icon'

**Comment by shivanraptor** [comment]
Same issue is also valid for `release-8.4` all the way to `release-8.8`

**Comment by xyunsh** [comment]
> How to add those icons. I open the icon sets in Xcode 13.0, but I found no missing icon placeholder. Or I have to update some Bazel `BUILD` file? Thanks.
> 
> p.s. the same issue happens in `release-8.3` p.s. the App without these recommended icons can still pass the App Store approval process.

Appreciate for any suggestion of this issue, Did you fixed this?

**Comment by xyunsh** [comment]
Same issue, any suggestion? thanks

**Comment by xyunsh** [comment]
```
ITMS-90892: Missing recommended icon - The bundle does not contain an alternate app icon for iPad of exactly '167x167' pixels, in .png format for iOS versions supporting iPad Pro. To support older operating systems, the icon may be required in the bundle outside of an asset catalog. Make sure the Info.plist file includes appropriate entries referencing the file. See https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleicons

ITMS-90892: Missing recommended icon - The bundle does not contain an alternate app icon for iPad of exactly '152x152' pixels, in .png format for iOS versions >= 10.0. To support older operating systems, the icon may be required in the bundle outside of an asset catalog. Make sure the Info.plist file includes appropriate entries referencing the file. See https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleicons

ITMS-90683: Missing purpose string in Info.plist - Your app’s code references one or more APIs that access sensitive user data, or the app has one or more entitlements that permit such access. The Info.plist file for the “Telegram.app” bundle should contain a NSLocationAlwaysAndWhenInUseUsageDescription key with a user-facing purpose string explaining clearly and completely why your app needs the data. If you’re using external libraries or SDKs, they may reference APIs that require a purpose string. While your app might not use these APIs, a purpose string is still required. For details, visit: https://developer.apple.com/documentation/uikit/protecting_the_user_s_privacy/requesting_access_to_protected_resources.

```

**Comment by xyunsh** [comment]
@ali-fareed can you help?
